src/modules/asab-config/MicroserviceDetailContainer.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import { 
+    Container, Card, CardBody, 
+    CardHeader, Col, Row
+ } from 'reactstrap';
+import { DateTime } from 'asab-webui';
+import ReactJSON from 'react-json-view';
+
+const MicroserviceDetailContainer = (props) => {
+    const [svc, setSvc] = useState(null);
+    const { t } = useTranslation();
+    const { svc_name } = useParams();
+
+    const ASABConfigAPI = props.app.axiosCreate('asab_config');
+
+    useEffect(() => {
+        ASABConfigAPI.get(`microservices/${svc_name}`)
+            .then(svc => setSvc(svc.data))
```

Comment:
@titkov49 Here it might be also good to catch the response from the config service. If there is some error message returned from the service on `200`. You want to check that with Mithun.